{"uuid": "49b023b9-f36b-4658-b73c-9e504dff4563", "children": ["8e477a12-b181-4c2b-bd65-39d8cdf46f7d", "38ccd19c-2306-4f15-a8a8-33b57c22e84c", "64019cd8-1f9c-478d-9a71-bae1171e7cf3", "e42f4fa5-5091-4591-9b3e-d8ed55f8603e", "b2f7a36b-535e-4fd5-9945-676bda244bae", "775f6f7c-5ef4-45b3-aacd-47e1fbe8f749"], "befores": [{"name": "setup_profiles", "status": "broken", "statusDetails": {"message": "TypeError: 'bool' object is not iterable\n", "trace": "  File \"/usr/local/lib/python3.8/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/wlan-testing/tests/e2e/basic/conftest.py\", line 79, in setup_profiles\n    return_var = fixtures_ver.setup_profiles(request, param, setup_controller, testbed, get_equipment_id,\n  File \"/wlan-testing/tests/fixtures_2x.py\", line 292, in setup_profiles\n    ap_wifi_data = ap_ssh.get_interface_details()\n  File \"/wlan-testing/tests/../libs/apnos/apnos.py\", line 396, in get_interface_details\n    for i in r:\n"}, "attachments": [{"name": "ucentral_config: ", "source": "c2b9d463-1e2b-4c01-82ee-4aedc7f78495-attachment.attach"}, {"name": "Time Took to apply Config: 243.96254801750183", "source": "61a715b4-8531-424a-8734-5c23cdcbd1bb-attachment.attach"}, {"name": "Success : Active Config in AP: ", "source": "6a9f7d02-19ba-467e-a6d3-0574c8a825fb-attachment.attach"}, {"name": "AP LOgs: ", "source": "cf4c8b44-f799-4034-abe3-29540ccc12b7-attachment.attach"}], "start": 1628989511365, "stop": 1628989831177}], "start": 1628989511364, "stop": 1628989831567}